cfpb/cfgov-refresh
# Fix JS error on /external-site

The header & footer scripts were initializing on /external-site even though it had no header or footer, which caused a JS error

## Changes

- Build scripts for external site separately

## Testing

- Check out branch & run gulp
- Navigate to http://localhost:8000/external-site/ and check that there is no js error related to missing header element

## Review

- @sebworks 
- @contolini 


## Screenshots

<img width="1268" alt="screen shot 2016-08-23 at 8 26 45 am" src="https://cloud.githubusercontent.com/assets/778171/17898236/ddbff196-690b-11e6-95f7-199bc35d8de8.png">

## Notes

-

## Todos

-

## Checklist
